app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,18 @@
+   <body>   
+    <div class='container'>
+      <h2>New Recipe:</h2>
+      <%= form_with model: @recipe, url: recipe_create_path, method: :post do |f| %>
+        <%= f.label :name %>
+        <%= f.text_field :name %>
+        <%= f.label :description %>
+        <%= f.text_area :description %>
+        <%= f.label "Preparation Time(hours):" %>
+        <%= f.number_field :preparation_time %>
+        <%= f.label "Cooking Time(hours):" %>
+        <%= f.number_field :cooking_time %>
+        <%= f.label :public %>
+        <%= f.check_box :public %>
+        <%= f.submit "Create Recipe" %>
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to make use of partial views to create your forms